### PR TITLE
some cleanup on working through the installation docs

### PR DESCRIPTION
**Pull request by patrickmmartin**

### CODE REVIEW

**Comment by mponomar** [comment]
I should simplify the instructions.  Specifying the database directory isn't necessary -- there's a reasonable default.

File: README.md [hunk 1]
```diff
@@ -103,33 +103,35 @@ needed.
...
-   comdb2 --lrl /home/mponomar/db/mikedb.lrl mikedb
+   comdb2 --lrl ~/db/mikedb.lrl mikedb
    ```
    All nodes will say 'I AM READY.' when ready.
 
 1. On any node, start using the database.  You don't have any tables yet.  You can add them with *cdb2sql* 
    Example -
    ```sql
-   cdb2sql mikedb local 'CREATE TABLE t1 {
+   ./cdb2sql mikedb local 'CREATE TABLE t1 {
```
Comment (mponomar) [inline README.md]: One of the steps is make install and add to PATH, so ./cdb2sql should work.  Like I said, these instructions can use an update.

Comment (patrickmmartin) [inline README.md]: ok will revert the assumption on PWD and check the PATH

**Comment by patrickmmartin** [comment]
re:

https://github.com/bloomberg/comdb2/pull/11#issuecomment-302255456

I presume omit - the `-dir` parameter in the command?

Also, what is that default, for future reference?

**Comment by mponomar** [comment]
[--dir command line argument](https://bloomberg.github.io/comdb2/args.html#using-dir)
`${COMDB2_ROOT}/var/cdb2/${dbname}`
For a default install, /opt/bb/var/cdb2/yourdb

File: README.md [hunk 2]
```diff
@@ -103,26 +103,27 @@ needed.
...
-   vi /home/mponomar/db/mikedb.lrl
+   vi ~/db/mikedb.lrl
    add
    cluster nodes mptest-1.comdb2.example.com mptest-2.comdb2.example.com
    ```
    
 9. On other nodes, copy the database over:
    ```
-   copycomdb2 mptest-1.comdb2.example.com:/home/mponomar/db/mikedb.lrl
+   copycomdb2 mptest-1.comdb2.example.com:~/db/mikedb.lrl
```
Comment (mponomar) [inline README.md]: `copycomdb2 centos:${HOME}/db/mikedb.lrl` works

`copycomdb2 centos:~/db/mikedb.lrl` won't be expanded by the shell (at least isn't by my shell)

**Pull request merged**